cpp/src/arrow/compute/kernels/CMakeLists.txt

```diff
@@ -126,7 +130,8 @@ add_arrow_compute_test(aggregate_test
                        SOURCES
                        aggregate_test.cc
                        EXTRA_LINK_LIBS
-                       arrow_compute_kernels_testing)
+                       arrow_compute_kernels_testing
```

Comment:
Should this be ${ARROW_COMPUTE_SCALAR_TYPE_TEST_LINK_LIBS} ?

Reply:
We don't need to use `${ARROW_COMPUTE_SCALAR_TYPE_TEST_LINK_LIBS}` (that is as list of `arrow_compute_kernels_testing` and `utf8proc::utf8proc`) here because `aggregate_test.cc` doesn't use utf8proc directly.